README.md

```diff
@@ -110,17 +110,16 @@ This plugin defines hooks for `environment`, `checkout`, `command`, and `post-co
 
 #### Available properties
 
-- `input_artifacts`: String array of artifact paths to download and sign.
+- `input_artifacts`: String array of artifact paths to download that are required to sign.
+- `sign_prerequisites`: String array of the specific artifact paths to sign. In the case of `.app`s, make sure to list these bottom-up; internal frameworks/helpers first, and `.app` at the end.
+- `output_artifacts`: String array of relevant artifact to upload back to artifacts.
```

Comment:
Does this need to be specified, or should this just be automatically derived from the input name of the thing that's signed?

Seems like it'd be simpler both for us and users that way (since they have to specify fewer params), and I can't think of why a user would _need_ us to transform a filename.

Reply:
Absolutely agree - buildkite download/upload semantics just aren't the same, and I wanted to get a working PR up first before I worried on YAML specifics.

But yeah: 
`download dir/*` will download all files and subdirectories,
`upload dir/*` will upload only files - not subdirectories.

Reply:
I mention the directory logic because `something.app` is a directory at the filesystem level, so requires directory handling in buildkite